content/2017-04-27_mi-pyt.md

```diff
@@ -0,0 +1,183 @@
+Title: Učili jsme Python na univerzitě, otevřeně a na GitHubu
+Date: 2017-04-27 19:00:00
+Modified: 2017-04-27 19:00:00
+Author: Petr Viktorin a Miro Hrončok
```

Comment:
Myslím, že by to mělo být `Petr Viktorin, Miro Hrončok` a pak se to chytí jako dva autoři. Což nevím, jestli se úplně nějak speciálně zobrazí, ale tuším, že jsem tam něco takového dělal.